.github/ISSUE_TEMPLATE/new-feature.yml

```
@@ -20,6 +20,12 @@ body:
     description: Describe the feature in one or two sentences.
   validations:
     required: true
+- type: textarea
+  attributes:
+    label: Documentation
+    description: Link(s) to documentation and/or demos of the feature.
```

Let's use the imperative, like the other fields:

```suggestion
    description: Link to documentation and demos of the feature.
```